### PR TITLE
feat: 🎸 InfoView

### PR DESCRIPTION
**Pull request by xiaoqinggrace**
Add InfoView. Modify the binding in ActivationScreen.

### CODE REVIEW

File: Sources/FioriSwiftUICore/Views/InfoView+View.swift [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                    .foregroundColor(.preferredColor(.primary1))
+                    .multilineTextAlignment(.center)
+            }
+        }
+        
+        struct ActionText: ViewModifier {
+            func body(content: Content) -> some View {
+                content
+                    .font(.system(size: 15))
+                    .frame(width: 169.0, height: 20.0)
```
Comment (billzhou0223) [inline Sources/FioriSwiftUICore/Views/InfoView+View.swift:41]: We want the frame of action button to be dynamic so probably set framework with `minWidth` and `minHeight` makes more sense.

File: Sources/FioriSwiftUICore/Views/InfoView+View.swift [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            descriptionText
+//                .padding(.bottom, 80)
+            Spacer()
+            progressIndicatorText
+//            if #available(iOS 14.0, *) {
+//                ProgressView()
+//                    .frame(width: 24, height: 24, alignment: /*@START_MENU_TOKEN@*/ .center/*@END_MENU_TOKEN@*/)
+//            } else {
+//                // Fallback on earlier versions
+//            }
```
Comment (billzhou0223) [inline Sources/FioriSwiftUICore/Views/InfoView+View.swift:92]: Let's clean up these commented code to make our code base cleaner.

File: Sources/FioriSwiftUICore/Views/WelcomeScreen+View.swift [hunk 3]
```diff
@@ -94,12 +106,19 @@ extension WelcomeScreen: View {
...
-                .padding(.top, 80)
+                .padding(.top, 60)
                 .padding(.bottom, 40)
             descriptionText
                 .padding(.bottom, 80)
+            
+            textFilled
+                .multilineTextAlignment(.center)
+                .keyboardType(.emailAddress)
+                .disableAutocorrection(true)
```
Comment (billzhou0223) [inline Sources/FioriSwiftUICore/Views/WelcomeScreen+View.swift:117]: Should these styles be set in textFilled's default view modifier?

**Review by billzhou0223 — CHANGES_REQUESTED** [review]
Ideally we don't want to include WelcomeScreen changes in this PR because this one is for InfoView. 

No need to worry about it this time but we should try to avoid this in the future.

File: Apps/Examples/Examples/FioriSwiftUICore/Onboarding/ActivationScreenSample.swift [hunk 4]
```diff
@@ -21,20 +22,35 @@ class ActivationScreenDataModel: ActivationScreenModel {
...
+            set: {
+                self.userInput = $0
+                model.updateValue(value: $0)
+                if $0.isEmpty {
+                    actionDisabled = true
+                } else {
+                    actionDisabled = false
+                }
+            }
+        )
```
[the remark this answers is not in the document]
Comment (xiaoqinggrace) [reply Apps/Examples/Examples/FioriSwiftUICore/Onboarding/ActivationScreenSample.swift:49]: It is for updating the parameter to be used in the didSelectAction(). We can sync in Slack.

File: Apps/Examples/Examples/FioriSwiftUICore/Onboarding/ActivationScreenSample.swift [hunk 5]
```diff
@@ -21,20 +22,35 @@ class ActivationScreenDataModel: ActivationScreenModel {
...
         VStack {
-            ActivationScreen(title: model.title_, descriptionText: model.descriptionText_, textFilled: model.textFilled_, actionText: model.actionText_, footnote: model.footnote_, secondaryActionText: model.secondaryActionText_, onCommit: model.onCommit, didSelectAction: model.didSelectAction, didSelectSecondaryAction: model.didSelectSecondaryAction)
+            ActivationScreen(title: model.title_, descriptionText: model.descriptionText_, textFilled: binding, actionText: model.actionText_, footnote: model.footnote_, secondaryActionText: model.secondaryActionText_, onCommit: model.onCommit, didSelectAction: model.didSelectAction, didSelectSecondaryAction: model.didSelectSecondaryAction)
```
[the remark this answers is not in the document]
Comment (xiaoqinggrace) [reply Apps/Examples/Examples/FioriSwiftUICore/Onboarding/ActivationScreenSample.swift:51]: The reason to use the current method for creation is due to the binding value textFilled: binding.

**Review by billzhou0223 — DISMISSED** [review]

**Comment by billzhou0223** [comment]
Resolved by #283

**Pull request closed without merging**